### ISSUE TITLE
SVO names for points on a river extracted from GRWL dataset

### ISSUE THREAD

**Issue by ankushkhandelwal** [mintproject/MINT-DataCatalog-Public#12]
I have created a dataset that contains locations of points on river. For example like this
<img width="555" alt="point_example" src="https://user-images.githubusercontent.com/36345336/62805231-fb081000-bab4-11e9-83f7-1be3add49744.png">

Each point will have a unique ID. I extracted this points from the GRWL dataset from here - 
https://doi.org/10.5281/zenodo.1297434

The unique IDs are created by me and are not taken from the dataset because this dataset provides information as a linestring. 

Please suggest a relevant name and uri

**Comment by mariutzica** [comment]
I am thinking this would be similar to issue #11 so I would say something like

basin~drainage_channel~river_point__khandelwal_identification_code

Is there some common attribute for all of the points? For example, do all of the points fall along the centerline of the river channel? Then the variable would be more specific, like:

basin~drainage_channel~river_centerline_point__khandelwal_identification_code

Where centerline = line~max-depth. What do you think?

**Comment by ankushkhandelwal** [comment]
Yes, according to the authors these polylines are center lines. I am fine with the name that you have suggested. Please add them to the ontology.

**Comment by mariutzica** [comment]
@ankushkhandelwal I just added the label ```basin~drainage_channel~river_centerline_point__khandelwal_identification_code``` to the ontology.
